Cargo.toml

```diff
@@ -44,6 +44,18 @@ unsafe_code = "forbid"
 
 [lints.clippy]
 unwrap_used = "deny"
+print_stdout = "deny"
+mut_mut = "deny"
+similar_names = "deny"
+unicode_not_nfc = "deny"
+enum_glob_use = "deny"
+if_not_else = "deny"
+items_after_statements = "deny"
+used_underscore_binding = "deny"
+panic = "deny"
+cast_possible_wrap = "deny"
+cast_possible_truncation = "deny"
+cast_sign_loss = "deny"
```

Comment:
Please order them lexicographically so it's a bit easier to see what's there.

Reply:
Updated.